mlflow/langchain/__init__.py

```diff
@@ -839,7 +820,8 @@ def predict(
         return result
 
 
-def _load_pyfunc(path):
+# TODO: Support loading langchain with model_config. For now, this is a no-op.
```

Comment:
@annzhang-db Do we have an internal ticket to track this? It would be very useful

Reply:
I'll make a ticket!